.semaphore/push-images/key-cert-provisioner.yml

```diff
@@ -0,0 +1,45 @@
+version: v1.0
```

Comment:
I basically copied from kube-controllers and replaced some text with key-cert-provisioner. I notice that all pipelines in calico are using ubuntu2004 now. I kept it the same.